config/locales/en.yml

```diff
@@ -1489,16 +1489,43 @@ en:
           weir: "Weir"
           "yes": "Waterway"
       admin_levels:
-        level2: "Country Boundary"
-        level3: "Region Boundary"
-        level4: "State Boundary"
-        level5: "Region Boundary"
-        level6: "County Boundary"
-        level7: "Municipality Boundary"
-        level8: "City Boundary"
-        level9: "Village Boundary"
-        level10: "Suburb Boundary"
-        level11: "Neighbourhood Boundary"
+        level2: "International Boundary"
+        level3: "Administrative Boundary (Level 3)"
+        level4: "Administrative Boundary (Level 4)"
+        level5: "Administrative Boundary (Level 5)"
+        level6: "Administrative Boundary (Level 6)"
+        level7: "Administrative Boundary (Level 7)"
+        level8: "Administrative Boundary (Level 8)"
+        level9: "Administrative Boundary (Level 9)"
+        level10: "Administrative Boundary (Level 10)"
+        level11: "Administrative Boundary (Level 11)"
```

Comment:
Those are longer prefixes, results often don't fit on one line and have line breaks in inconvenient places:

![image](https://github.com/user-attachments/assets/f2e98221-0b3b-4b2a-b9d1-09d1cb4ae109)

Reply:
I noticed that other boundary types don’t say “Boundary”. For example, `boundary=electoral` is just “Electoral”. We could shorten “Administrative Boundary (Level 4)” to just “Administrative (Level 4)”, but I’m ambivalent about similarly truncating the `border_type=*`-based labels, such as “City ~~Boundary~~”. That would easily get confused with the `place=*`-based labels, which follow different criteria.

Another option would be to drop the level suffix, since @tomhughes doesn’t seem enamored with including this information anyways. To the extent that this introduces ambiguity in the search results, the solution would be to tag boundaries with `border_type=*`, which is a good idea anyways.

Reply:
> I noticed that other boundary types don’t say “Boundary”. For example, boundary=electoral is just “Electoral”. 

Those are constructed here:
https://github.com/openstreetmap/openstreetmap-website/blob/5d76ec051e2c429b6647401674e13688c1251956/app/assets/javascripts/index/query.js#L91-L100

Reply:
I thought about using icons in search/query result lists, like on changeset/element pages. For administrative boundaries it could be something like a square with a border similar to the boundary line rendered on the map, maybe with a number inside for the level.

Reply:
Not a bad idea overall, though the numbers probably aren’t as intuitive for many mappers as designations coming from either `border_type=*` or `place=*`. I had forgotten about the code to transform an unrecognized tag into a label. As long as we have labels, we could fall back to a format string like “_Border Type_ Boundary” if `border_type=*` is present but set to an unrecognized value.

Reply:
Numbers tell how relatively big/important the area is, something that may not be clear otherwise.

Reply:
Would the number be clearer than the locally common word for the type of boundary? Mappers who specialize in boundaries would know the local `admin_level=*` scale by heart, but others would find words more intuitive and would already know which are larger than others. Also, the Overpass results are ranked by size.